### ISSUE TITLE
puppeteer_dev_chrome_profile-x

### ISSUE THREAD

**Issue by 0xDevrim** [astefanutti/decktape#295]
After the recent update, we've started to get hundreds of these profiles in the `tmp` directory which takes up the whole storage on our worker servers. There is an issue on `puppeteer` (https://github.com/puppeteer/puppeteer/issues/6414). Is there anything we can do via decktape? Is the browser not being closed via decktape or something?

**Comment by astefanutti** [comment]
Thanks for the report. There is indeed mentions that missing `browser.close()` call could be the issue, but I think Decktape does that already.

There are other mention that one work-around could be do add the following parameter:
```
const browser = await  puppeteer.launch({
  userDataDir: '/dev/null'
});
```

That being said, that may likely be a regression in Puppeteer, that'll hopefully be fixed in a newer version.

**Comment by astefanutti** [comment]
This should be fixed with #321. Thanks for your patience.
